packages/central-server/pm2.config.cjs

```diff
@@ -29,8 +29,9 @@ function task(name, args, instances = 1, env = {}) {
     exec_mode: 'fork',
     restart_delay: 5000,
     env: {
-      NODE_ENV: 'production',
       ...env,
+      NODE_ENV: 'production',
```

Comment:
Noticed that it's currently possible to override the `NODE_ENV` with the pm2 env. @chris-bes and I were chatting and that felt wrong, so changed this here. But perhaps @edmofro or @passcod you know if this was intentional?

Reply:
I guess if you run pm2 locally for some reason? No objection from me to change it

Reply:
Nothing I can think of / remember